### PR TITLE
Moved the sidebar settings to the appearance settings page

### PR DESCRIPTION
**Pull request by yaichenbaum**
Future PRs will
- restyle the navigation view
- restyle the setting blocks to closer match the Windows settings app
- merge and better organize more setting pages

### CODE REVIEW

**Comment by d2dyno1** [comment]
Could you share a screenshot?

**Comment by yaichenbaum** [comment]
image

**Comment by d2dyno1** [comment]
I'm concerned about the text clipping. While it looks fine in English, it may not look as good in German for instance.

**Comment by yaichenbaum** [comment]
> I'm concerned about the text clipping. While it looks fine in English, it may not look as good in German for instance.

This PR fixes the text clipping in the setting blocks that happened in previous versions. If you're referring to the navigation view but that will be restyled in phase 2.

**Comment by gave92** [comment]
I suppose I shouldn't care about the close button overlapping the "About" item right?

File: Files/Dialogs/SettingsDialog.xaml.cs [hunk 1]
```diff
@@ -29,26 +29,13 @@ private void Current_SizeChanged(object sender, WindowSizeChangedEventArgs e)
...
-            {
-                SettingsPane.PaneDisplayMode = Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Left;
-                SettingsContentFrame.Width = 460;
-                Column0.Width = new GridLength(0, GridUnitType.Auto);
-            }
-
             if (Window.Current.Bounds.Height <= 600)
             {
-                ContainerGrid.Height = Window.Current.Bounds.Height;
+                SettingsScrollViewer.Height = Window.Current.Bounds.Height - 40;
```
Comment (gave92) [inline Files/Dialogs/SettingsDialog.xaml.cs:34]: Curious: why is it necessary to manually set the ScrollViewer height?

**Comment by yaichenbaum** [comment]
> I suppose I shouldn't care about the close button overlapping the "About" item at this point right?

Pretend it's not there 😉

File: Files/Dialogs/SettingsDialog.xaml.cs (hunk 1 above)
Comment (yaichenbaum) [reply Files/Dialogs/SettingsDialog.xaml.cs:34]: It's the only way I was able to get it to stretch.

**Review by gave92 — APPROVED** [review]

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**